deployments/helm/hephaestus/values.yaml

```diff
@@ -117,6 +117,10 @@ controller:
   # Additional volume mounts added to both the manager and vector containers
   extraVolumeMounts: []
 
+  ibGarbageCollector:
```

Comment:
Nitpicking again, `ibGarbageCollector` isn't the clearest name - I'd rather see something verbose like `completedImageBuildCleanup`.